inc/ticket.class.php

```diff
@@ -4865,18 +4867,60 @@ function showForm($ID, $options = []) {
          $content = Html::cleanPostForTextArea($content);
       }
 
-      $content = Html::setRichTextContent(
-         $content_id,
-         $content,
-         $rand,
-         !$canupdate
-      );
+      Html::initEditorSystem($content_id, $rand, true, !$canupdate);
+      if ($ID > 0) {
+         $content = Html::clean($content, false, 1);
+      }
+
+      // If content does not contain <br> or <p> html tag, use nl2br
+      if (!preg_match("/<br\s?\/?>/", $content) && !preg_match("/<p>/", $content)) {
+         $content = nl2br($content);
+      }
 
       echo "<div id='content$rand_text'>";
       if ($canupdate || $can_requester) {
-         echo "<textarea id='$content_id' name='content' style='width:100%' rows='$rows'".
-               ($tt->isMandatoryField('content') ? " required='required'" : '') . ">" .
-               $content."</textarea></div>";
+         Html::textarea([
+            'name'            => 'content',
+            'editor_id'       => $content_id,
+            'required'        => $tt->isMandatoryField('content'),
+            'rows'            => $rows,
+            'enable_richtext' => true,
+            'value'           => $content,
+         ]);
+         if (isset($this->input['_content'])) {
```

Comment:
in this if() is displayed the uploaded file names and a button to remove them (see fileupload.js, function displayUploadedFile()